x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/DateScriptFieldScript.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Map;
 
 public abstract class DateScriptFieldScript extends AbstractLongScriptFieldScript {
-    public static final ScriptContext<Factory> CONTEXT = new ScriptContext<>("date_script_field", Factory.class);
+    public static final ScriptContext<Factory> CONTEXT = newContext("date", Factory.class);
```

Comment:
I've only done this to a single script field. If we like it I'd be happy to do it in a follow up.